front/lib/models/assistant/actions/retrieval.ts

```diff
@@ -105,6 +112,15 @@ AgentRetrievalConfiguration.init(
             );
           }
         }
+
+        // Validation for TopK
+        if (retrieval.topKMode == "custom") {
```

Comment:
this will fail in the future if we don't run a migration to remove topK from the existing assistants?

Reply:
Added a migration script